doc/geany.txt

```diff
@@ -2776,14 +2776,35 @@ The project menu items are detailed below.
 New project
 ^^^^^^^^^^^
 
-To create a new project, fill in the *Name* field. By default this
-will setup a new project file ``~/projects/name.geany``. Usually it's
-best to store all your project files in the same directory (they are
-independent of any source directory trees).
-
-The Base path text field is setup to use ``~/projects/name``. This
-can safely be set to any existing path -- it will not touch the file
-structure contained in it.
+There are two ways of creating new projects, either by using
+*Project->New* menu item or by using *Project->New from Folder* menu
+item.
+
+New
+    This method is more suitable for creating new, empty projects from
+    scratch at the default location without having any existing sources.
+
+    To create a new project, fill in the *Name* field. By default this
+    will setup a new project file ``~/projects/name/name.geany``.
+
+    The Base path text field is setup to use ``~/projects/name``. This
```

Comment:
Should Base or Base path be *ed like Name above?  If so also doesn't need "text", you don't put field type on any other field.

Reply:
The `*`s weren't present before but yeah, better to use them here so I added them for all the field names.